packages/office-ui-fabric-react/src/components/Button/ButtonBase/ButtonBase.tsx

```diff
@@ -0,0 +1,87 @@
+/* tslint:disable:no-unused-variable no-unused-imports */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable no-unused-imports*/
+import {
+  BaseComponent,
+  css,
+  assign,
+  getId,
+  getNativeProps,
+  buttonProperties,
+  anchorProperties
+} from '../../../Utilities';
+import { IButtonProps, IButton } from './ButtonBase.Props';
+import './ButtonBase.scss';
```

Comment:
this shouldn't be there... I think this should maybe be on Button only. There's a big assumption that all buttons have those rules, which is not true.